src/model/regex/structures/RegexCharacterClassInverted.ts

```diff
@@ -0,0 +1,25 @@
+class RegexCharacterClassInverted extends RegexCharacterClass {
+  constructor(charSet: string) {
+    super(charSet)
+    // if (charSet.length == this.chars.size)
+    //   this.chars.delete(getRandomElementFromArray(charSet.split("")))
```

Comment:
Why commented out?

Reply:
This would ensure that there is at least one possible answer.
Without it, it could happen that there is an inverted character class which cant have any answer (or only the empty answer)
I don't know if this would be a good thing, as it does work without it, but maybe it would be nicer if there is always at least one possible option.

Reply:
No it wouldn't be game breaking, this character class would just evaluate to an empty string
eg `charset=["a","b"]` and `regex=a*[^ab]b*` would just equal `regex=a*b*`